backend/internal/model/game.go

```diff
@@ -36,16 +36,16 @@ type GameScore struct {
 }
 
 type GameStats struct {
-	WhiteShotsCount int
-	BlueShotsCount  int
-	FastestShot     Shot
-	ManualGoals     map[int]map[string]int
-	Heatmap         [config.HeatmapAccuracy][config.HeatmapAccuracy]int
+	WhiteShotsCount int                                                 `json:"whiteShotsCount"`
+	BlueShotsCount  int                                                 `json:"blueShotsCount"`
+	FastestShot     Shot                                                `json:"fastestShot"`
+	ManualGoals     map[int]map[string]int                              `json:"manualGoals"`
+	Heatmap         [config.HeatmapAccuracy][config.HeatmapAccuracy]int `json:"heatmap"`
```

Comment:
Maybe a type alias could be prettier for those 2 long types?

Reply:
We will consider it.